pkg/endpoint/log.go

```diff
@@ -131,7 +131,7 @@ func (e *Endpoint) UpdateLogger(fields map[string]interface{}) {
 func (e *Endpoint) updatePolicyLogger(fields map[string]interface{}) {
 	policyLogger := e.policyLogger.Load()
 	// e.Options check needed for unit testing.
-	if policyLogger == nil && e.Options != nil && e.Options.IsEnabled(option.DebugPolicy) {
+	if policyLogger == nil && e.Options != nil {
```

Comment:
Looks like this change is not needed due to the above change to use `e.getLogger()` instead?